index.bs

```diff
@@ -840,6 +840,7 @@ When the {{MLContext/[[contextType]]}} is set to [=context type/default=] with t
   <summary>
     To <dfn>validate graph resources</dfn>, given {{MLNamedArrayBufferViews}} |resources| and [=ordered map=] |descriptors|, run the following steps:
   </summary>
+    1. If |resources|'s [=map/size=] does not equal |descriptors|'s [=map/size=], then return false.
```

Comment:
Isn't this implied by the loop below? (but a quick test is fine)

Reply:
If _resources_'s keys are [ "a", "b" ] but _descriptors_'s keys are [ "a", "b", "c" ] the loop below won't fail:

> 1. For each name → resource of resources:

So this would loop over [ "a", "b" ]

>> 1. If descriptors[name] does not exist, return false.

_descriptors_["a"] and _descriptors_["b"] exist, so this does not fail

>> 2. If validating buffer with descriptor given resource and descriptors[name] returns false, then return false.

Presumably this also doesn't fail

> 2. Return true.

So we return true, even though "c" is not provided.